x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_unfollow_provider.js

```diff
@@ -89,9 +89,9 @@ class Provider extends PureComponent {
               <p>
                 <FormattedMessage
                   id="xpack.crossClusterReplication.unfollowLeaderIndex.confirmModal.singleUnfollowDescription"
-                  defaultMessage="This follower index will be paused, closed, and converted into a regular index.
-                    The index will no longer appear in Cross Cluster Replication but can still be managed using Index
-                    Management. This operation cannot be undone."
+                  defaultMessage="This follower index will be converted to a standard index and no
+                    longer appear in Cross Cluster Replication, but you can manage it in Index
```

Comment:
The follower index will be converted to a standard index.  It will no longer appear in ...